consensus/istanbul/core/commit.go

```diff
@@ -45,7 +45,33 @@ func (c *core) generateCommittedSeal(sub *istanbul.Subject) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	return committedSeal, nil
+	return committedSeal[:], nil
```

Comment:
Should the `generateCommittedSeal` function also return a `blscrypto.SerializedSignature` type, as opposed to a `[]byte` type?